.github/workflows/docker-build-and-publish-linux-amd64.yml

```
@@ -0,0 +1,27 @@
+name: Build and Publish Linux AMD64 Image
+
+on:
+  workflow_dispatch: # allows us to trigger runs manually in the GitHub UI
+  schedule:
+    - cron:  '0 0 * * *' # runs nightly
+
+jobs:
+  docker:
+    if: ${{ github.ref == 'refs/heads/main' }}
+    runs-on: ubuntu-latest
```

since we're pinning to `ubuntu-24.04-arm64` for arm64, maybe we pin to `ubuntu-24.04` for consistency?

```suggestion
    runs-on: ubuntu-24.04
```

---

Was only using `ubuntu-24.04-arm64` because there's no `ubuntu-latest-arm64` (not sure why 🤔) but I think we can be more consistent on the other side..